### DISCUSSION TITLE
[해결] JsEnv는 프로덕션에서 사용할 수 없나요?

### DISCUSSION THREAD

**Discussion by runasy-koonta** [naverz/zepeto-studio-kor#296] [5. Q&A]
기본적으로 제공되지 않는 Delegate(Vector를 Parameter로 받는 action)을 사용하기 위해
JsEnv의 UsingAction을 사용하였는데요,

로컬 및 QR Code를 스캔해 모바일에서 했을 땐 잘 되었는데
Publish한 뒤 테스트 링크를 통해 접속하니 작동하지 않네요.

아무래도 프로덕션이라 어느 에러가 발생하는지 알 수가 없어서,
이걸 의도적으로 막아놓으신건지 궁금합니다.

**Comment by runasy-koonta** [comment]
자문자답입니다,

JsEnv를 사용할 수 있었는데
서버 커넥션 부분 다른 부분이 문제가 있어서 안 되던 거였습니다 ㅠㅠ

import { JsEnv } from 'Puerts'; 를 통해 사용 가능합니다!

**Reply by mayko1234** [reply]
정보 공유해주셔서 감사합니다😊
